ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/AppLovinUtils.java

```diff
@@ -197,10 +197,13 @@ public static AppLovinAdSize appLovinAdSizeFromAdMobAdSize(@NonNull Context cont
     ArrayList<AdSize> potentials = new ArrayList<>();
     potentials.add(AdSize.BANNER);
     potentials.add(AdSize.LEADERBOARD);
+    potentials.add( AdSize.MEDIUM_RECTANGLE );
```

Comment:
Just fyi, we have to match Google's conventions since this is their codebase
```suggestion
    potentials.add(AdSize.MEDIUM_RECTANGLE);
```

Reply:
Right, muscle memory 🤦